packages/vscode-extension/src/chat/utils.ts

```diff
@@ -13,7 +13,13 @@ export async function verbatimCopilotInteraction(
   response: ChatResponseStream,
   token: CancellationToken
 ) {
-  const chatRequest = await lm.sendChatRequest(model, messages, {}, token);
+  const [vendor, family] = model.split(/-(.*)/s);
```

Comment:
Is it better to change the `model` parameter in signature?

Reply:
Yeah, I think the better way is to manage the models together. This PR is to do the minimal change for unblocking our chat.